onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-result-types-pages/innovation-dev-info/components/scale-impact-analysis/scale-impact-analysis.component.html

```diff
@@ -7,7 +7,7 @@
   [verticalAlignment]="true" [options]="this.options?.responsible_innovation_and_scaling?.q2?.options"
   optionLabel="question_text" optionValue="result_question_id" subLabel="Multiple answers can be selected."
   [checkboxConfig]="{ listAttr: 'subOptions', optionLabel: 'question_text', optionValue: 'answer_boolean', optionTextValue: 'answer_text'}"
-  [(ngModel)]="this.options?.responsible_innovation_and_scaling?.q2['radioButtonValue']">
+  [(ngModel)]="this.options.responsible_innovation_and_scaling.q2['radioButtonValue']">
```

Comment:
@xKeCo any reason why the null checks were removed?

Reply:
@gcmartinezg After upgrading Angular from version 14 to 18, the optional chaining operators started causing errors when saving information in this section. I'm not entirely sure why, but it seems to be related to changes introduced in the newer Angular versions.